### DISCUSSION TITLE
sauce service - links to the sauce labs jobs are missing in the test report

### DISCUSSION THREAD

**Discussion by alexStrusberg** [webdriverio/webdriverio#7057] [Q&A]
Hi,

I am running with sauce labs and I saw that if the spec file has a few ITs then the final report includes only a single link to the job at sauce labs. why is that ? (e.g. spec reporter)

Thank you,
Alex S

**Comment by christian-bromann** [comment]
Every spec file runs within its own worker and creates its own WebDriver session. That means that all `it` blocks are part of that one session and therefor there is just one link to one job.
